styles.css

```
@@ -412,7 +455,257 @@ margin: 22px 0;
+}
+
+.global-section .card-container .card h3 {
+border-left:  2px solid  aqua;
+padding-left: 8px;
+line-height: 28px;
+}
+.global-section a {
+  color: aqua;
+}
+.two-text {
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: flex-start;
+  box-sizing: border-box;
+}
+.two-text .text-left {
+  width: 50%;
+  padding-right: 16px;
+  display: flex;
+  flex-direction: column;
+  align-items: start;
+  justify-content: center;
+  box-sizing: border-box;
+}
+.two-text .text-right {
+  width: 50%;
+  padding-left: 16px;
+  display: flex;
+  flex-direction: column;
+  align-items: start;
+  justify-content: center;
+  box-sizing: border-box;
+}
+/* Sexta seccion */
+
+.virtual_section {
+  display: flex;
+  justify-content: center;
+  padding: calc(64px * 2) 0;
+  position: relative;
+}
+
+.virtual_section::before {
+  content: "";
+  background-color:white;
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  top: 0;
+  z-index: -1;
+}
+
+
+.virtual_section .text-left {
+ 
+  padding-right:64px ;
+}
+
+.swipe_sessions {
+  width: 540px;
+  height: 446px;
+  box-shadow: 0px 2px 40px rgba(0, 0, 0, 0.15);
+  border-radius: 20px;
+  overflow: hidden;
+}
+
+.swipe_sessions .card_top {
+  width: 100%;
+  height: 50%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+}
+.swipe_sessions .card_top::before {
+  content: "";
+  background: linear-gradient( 101.33deg, #08209a 0.76%, #6563ff 33.33%, #36c5f0 76.92%, #83e2ff 96.96% );
+  width: 100%;
+  height: 150%;
+  position: absolute;
+  top: -40%;
+  transform: skewY(-5deg);
+  z-index: 1;
+}
+
+#sessions_logo {
+  z-index: 2;
+  width: 338px;
+}
+
+.virtual_section .avatars {
+  position: absolute;
+  bottom: -70px;
+  left: 64px;
+  z-index: 3;
+}
+
+.virtual_section .avatars img {
+  width: 80px;
+}
+.virtual_section .avatars img:nth-of-type(2) {
+  position: absolute;
+  left: 55px;
+}
+
+.virtual_section .card_bottom {
+  width: 100%;
+  height: 50%;
+  box-sizing: border-box;
+  padding: 64px;
+}
+.virtual_section .card_bottom .unified-text {
+  margin-top:32px;
+}
+
+.virtual_section .card_bottom .card-text {
+  color: black;
+  font-weight: 700;
+  width: 50%;
+}
+
+.waiting_section {
+  display: flex;
+  justify-content: center;
+  padding: 58px 0;
+}
+
+.waiting_section .text-left {
+  padding: 0 64px 0 0;
+}
+
+.waiting_section .text-right {
+  flex-direction: row;
+}
+
+.waiting_section .text-right .card {
+  width: 50%;
+  box-sizing: border-box;
+  padding: 0 8px 0 0;
+  margin: 0 16px 0 0;
+}
+
+.waiting_section .start-button {
+  margin-bottom: 34px;
+}
+
+.waiting_section h3 {
+  font-size: 20px;
+}
+.waiting_section .card-text {
+  color: #0a2540;
+}
+
+
+/* footer */
+
+footer {
+  font-family:"Anek Telugu", sans-serif;
+  display: flex;
+  justify-content: center;
+  background-color: rgba(220, 218, 218, 0.187);
+  padding: 64px 0;
+}
+footer .section-container {
+  display: flex;
```

Una posible solución podría ser darle un width al contenedor...
```suggestion
  display: flex;
  width: 90%;
```